### ISSUE TITLE
New slack emoji upload form

### ISSUE THREAD

**Issue by robin31** [jmhobbs/cultofthepartyparrot.com#259]
It seems slack had a new form for uploading.
This gives an exception with the python script.

```
Traceback (most recent call last):
  File "upload_parrots.py", line 112, in <module>
    sys.exit(main())
  File "upload_parrots.py", line 107, in main
    uploader.upload(parrot)
  File "upload_parrots.py", line 61, in upload
    (By.ID, "custom_emoji")))
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/support/wait.py", line 80, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
```

New form:
![image](https://user-images.githubusercontent.com/4686898/44979146-9a9b5500-af6c-11e8-8985-7558ec6a0739.png)

**Comment by nuragic** [comment]
`:sadparrot:`

Just my 2cents... perhaps it would be easier to do it with https://github.com/GoogleChrome/puppeteer ?

**Comment by nuragic** [comment]
Fixed in #264 `:opensourceparrot:`

**Comment by jmhobbs** [comment]
Woo hoo!
